ste/xfer-anyToRemote-folder.go

```diff
@@ -82,7 +82,7 @@ func anyToRemote_folder(jptm IJobPartTransferMgr, info *TransferInfo, pacer pace
 	} else {
 
 		t := jptm.GetFolderCreationTracker()
```

Comment:
is there any way we can test this by mocking the storage clients?

Reply:
Not sure. Will need to check